src/cloudkit.cs

```diff
@@ -350,6 +555,52 @@ interface CKFetchRecordChangesOperation {
 			set; 
 		}
 	}
+	
+	[iOS (10,0), Watch (3,0), TV (10,0), Mac (12,10, onlyOn64 : true)]
+	[BaseType (typeof(CKDatabaseOperation))]
+	interface CKFetchRecordZoneChangesOperation
+	{
+		[Export ("initWithRecordZoneIDs:optionsByRecordZoneID:")]
+		IntPtr Constructor (CKRecordZoneID[] recordZoneIDs, [NullAllowed] NSDictionary<CKRecordZoneID, CKFetchRecordZoneChangesOptions> optionsByRecordZoneID);
+
+		[Export ("recordZoneIDs", ArgumentSemantic.Copy)]
+		CKRecordZoneID[] RecordZoneIds { get; set; }
+
+		[NullAllowed, Export ("optionsByRecordZoneID", ArgumentSemantic.Copy)]
+		NSDictionary<CKRecordZoneID, CKFetchRecordZoneChangesOptions> OptionsByRecordZoneId { get; set; }
```

Comment:
Then we have an issue. I found the following methods already in the bindings:
 
    

- FetchUserRecordId
- CKRecordID CKDiscoveredUserInfo.UserRecordId { get; }
- CKRecordID [] CKDiscoverUserInfosOperation.UserRecordIds { get; set; } 
- NSString CKErrorFields.PartialErrorsByItemIdKey { get; }

And many others added by @dalexsoto in the first (at least 2014-06-10) bindings. I was just following the pattern already in the file. I can alway rename ALL of them and done a change to land in XAMCORE_4_0 or add them and deprecate the already present ones.